types/braintree-web/modules/hosted-fields.d.ts

```diff
@@ -43,7 +43,7 @@ export interface HostedFieldFieldOptions {
     expirationDate?: HostedFieldsField | undefined;
     expirationMonth?: HostedFieldsField | undefined;
     expirationYear?: HostedFieldsField | undefined;
-    number: HostedFieldsField;
+    number?: HostedFieldsField;
```

Comment:
Please update this

```suggestion
    number?: HostedFieldsField | undefined;
```

Reply:
Done: abaead92af29b1b4f0009347a857000e61e9c4a9